active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,324 @@
+
+![A sketch of five horizontal dots with the first selected and the previous and next buttons are visible](images/PipsPager_Visible.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VisibleButtonPipsPager"
+  NumberOfPages="5"
+  PreviousButtonVisibility=Visible
+  NextButtonVisibility=Visible />
+```
+
+#### Vertical PipsPager with VisibleOnPointerOver button visibility
+
+The PipsPager can also be oriented vertically which does not change its behavior or interaction.
+The various button visibility modes can apply to either orientation.
+The below example highlights the VisibleOnPointerOver button visibility. This button visibility 
+
+![A prototype of five vertical dots with the second initially selected and the previous and next buttons appearing and disappearing based on user hover and the selected page](images/PipsPager_VisibleOnPointerOver.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VerticalPipsPager"
+  NumberOfPages="5"
+  Orientation=Vertical 
+  PreviousButtonVisibility=VisibleOnPointerOver 
+  NextButtonVisibility= VisibleOnPointerOver />
+```
+
+#### Number of pages displayed less than total
+
+If the number of pages of content is large and do not need to be navigated to at once,
+the MaxVisiblePips property can be set to limit the number of visible, interactable, pips.
+If the NumberOfPages is greater than the MaxVisiblePips then the pips will scroll so that the selected page is
+centered in the control. If the NumberOfPages is equal to or less than the MaxVisiblePips,
+no scrolling will occur and the number of pips shown will be the same as the NumberOfPages.
+
+By default, the number of visible pips is 5.
+
+![A prototype of twelve horizontal dots with the first initially selected and the dots scrolling when the subsequent pages are selected](images/PipsPager_MaxVisiblePips.gif)
+
+XAML
+```XAML
+<muxc:PipsPager x:Name="ScrollingPipsPager"
+  NumberOfPages="20"
+  MaxVisiblePips="10" />
+```
+
+#### Integrate PipsPager with a Collection control
+
+Commonly, a PipsPager is used in conjunction with a Collection control.
+The below example shows how to bind a PipsPager with a FlipView to provide
+an extra way to navigate through content and indicate its page.
+
+Note: To use the PipsPager as a page indicator **only**,
+set IsEnabled to false in the control to disable user interaction.
+
+![A prototype of five dots, the PipsPager, laid out horizontally underneath a FlipView showing a picture of a bicycle. The third dot in the PipsPager is selected indicated the 3rd page of content.](images/PipsPager_FlipViewExample.png)
+
+XAML
+``` XAML
+<StackPanel>
+  <FlipView x:Name="Gallery" MaxWidth="400" Height="270" ItemsSource="{x:Bind Pictures}">
+      <FlipView.ItemTemplate>
+          <DataTemplate x:DataType="x:String">
+              <Image Source="{x:Bind Mode=OneWay}"/>
+          </DataTemplate>
+      </FlipView.ItemTemplate>
+  </FlipView>
+
+  <muxc:PipsPager x:Name="FlipViewPipsPager"
```

Comment:
Is PipsPager hard-coded to understand FlipView somehow? Or is code required here to tie these together?

Reply:
No it is not tied together. The two way binding on SelectedIndex is what makes that work.

Reply:
Ah, sorry, I missed that. Thanks